src/templates/success/success.template.html

```diff
@@ -0,0 +1,836 @@
+                                        </td>
+                                      </tr>
+                                    </tbody>
+                                  </table>
+                                </td>
+                              </tr>
+                              <div class="dark-img" style="mso-hide: all; display: none; padding:10px 25px;">
+                                <img src="../assets/logo_pagopa_white.png" width="72" class="dark-img" style="display: none" />
+                              </div>
+                              <tr>
+                                <td class="spacer" style="font-size:0px;word-break:break-word;">
+                                  <div style="height:8px;line-height:8px;">&#8202;</div>
+                                </td>
+                              </tr>
+                              <tr>
+                                <td align="left" class="title" style="font-size:0px;padding:10px 25px;word-break:break-word;">
+                                  <div style="font-family:Titillium Web, system-ui, sans-serif;font-size:13px;font-weight:bold;line-height:1;text-align:left;color:#000000;">
+                                    <h1 style="font-size: 32px; line-height: 40px; margin: 0;"> Hai inviato un pagamento di {{transaction.amount}}&nbsp;&euro; tramite pagoPA </h1>
+                                  </div>
+                                </td>
+                              </tr>
+                              <tr>
+                                <td class="spacer" style="font-size:0px;word-break:break-word;">
+                                  <div style="height:8px;line-height:8px;">&#8202;</div>
+                                </td>
+                              </tr>
+                              <tr>
+                                <td align="left" class="text" style="font-size:0px;padding:10px 25px;word-break:break-word;">
+                                  <div style="font-family:Titillium Web, system-ui, sans-serif;font-size:16px;font-weight:regular;line-height:24px;text-align:left;color:#000000;">
+                                    <p> Il tuo pagamento &egrave; andato a buon fine. Ecco i dettagli dell’operazione. </p>
+                                  </div>
+                                </td>
+                              </tr>
+                            </tbody>
+                          </table>
+                        </div>
+                        <!--[if mso | IE]></td></tr></table><![endif]-->
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </div>
+              <!--[if mso | IE]></td></tr></table></td></tr><![endif]-->
+              {{#eq cart.items.length 1}}
```

Comment:
To use `eq`, is to be introduced  custom helper or something like https://www.npmjs.com/package/handlebars-helpers

Reply:
@infantesimone That's right. We currently use two custom helpers: `eq` (equivalent to `a === b`) and `not` (equivalent to `a !== b`). We can add these custom helpers ([which you can find in this repo](https://github.com/pagopa/pagopa-email-templates/tree/main/Receipt_Success/helpers)) or directly use [handlebars-helpers](https://www.npmjs.com/package/handlebars-helpers) as you suggested.

Reply:
Solved with [42f6d3c](https://github.com/pagopa/pagopa-notifications-service/pull/15/commits/42f6d3ce745c7520ae50ebd47ccf1c089bed3af8)